external/webserver/http_client.h

```diff
@@ -52,6 +52,10 @@ struct http_client_t {
 #endif
 
 	int keep_alive;
+	int keep_alive_timeout;
+	int max_request;
```

Comment:
If `max_request` is not greater than `MAX_CLIENT_REQUEST(999999)`,
how about using `int16_t` or `uint16_t` for `max_request` and `remaining_request`?
Then we can save the memory.

Reply:
ok changes will be made

Reply:
2^16 = 65,536